indico/modules/events/papers/client/js/util.js

```diff
@@ -0,0 +1,13 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+export const PaperState = {
+  accepted: 'accepted',
+  rejected: 'rejected',
+  submitted: 'submitted',
+  toBeCorrected: 'to_be_corrected',
```

Comment:
Not sure about `camelCase` vs `snake_case` here... being able to search for the state name and find both backend and frontend code would be pretty convenient...